openshift/templates/nodejs.json

```diff
@@ -171,21 +185,17 @@
   "parameters": [
     {
       "name": "SOURCE_REPOSITORY_URL",
-      "description": "The URL with your Node.js application source code",
+      "description": "The URL of the repository with your application source code",
       "value": "https://github.com/openshift/nodejs-ex.git"
     },
     {
-      "name": "FRONTEND_SERVICE_NAME",
```

Comment:
What is the service name stuff removed? I think just missed due to timing of pr #5